### PR TITLE
feat: add hex to initial tokens

### LINKED ISSUE #563: Token request: Adding HEX token
**Please provide the following information for your token.**

Token Address: 0x2b591e99afE9f32eAA6214f7B7629768c40Eeb39
Token Name (from contract): HEX
Token Decimals (from contract): 8
Token Symbol (from contract): HEX
Uniswap Exchange Address of Token: 0x05cDe89cCfa0adA8C88D5A23caaa79Ef129E7883

Link to the official homepage of token: https://hex.win/
Link to CoinMarketCap page of token: https://coinmarketcap.com/currencies/hex/

Some tokens (e.g. BNB) do not work with Uniswap v1. In order to assess if your token works correctly, please complete small-value transactions of each of the types below, and submit the Etherscan transaction links for our review.
Test `addLiquidity` transaction: https://etherscan.io/tx/0xf8b502dfe08888883c026a4d78a2e764d39df9dd9e40b640f5ed95721893f4e8
Test `swap` transaction: https://etherscan.io/tx/0x386cfc7d441d476f49f21eb7faeb3dfb51dcc2d47dac62d87ba4bd114bb98fb6
Test `removeLiquidity` transaction: 
https://etherscan.io/tx/0x0ae4a79be678a47cb8b80b0a96b6b7fceb8431636cfb402b037b50ba949b7a6e

Are you willing to add liquidity to the liquidity pool for this token? (Y/N):  Yes
If so, how much liquidity are you willing to add?: up to 10 ETH

### PR DESCRIPTION
**Pull request by Justbeingjustin**
Fixes #563 

**Intentions:**
Navigate to https://uniswap.exchange/swap
Click "Select a token".
My intention of this PR is to add Hex to this drop down list.

### CODE REVIEW

File: src/contexts/Tokens.js [hunk 1]
```diff
@@ -201,6 +201,12 @@ export const INITIAL_TOKENS_CONTEXT = {
       [SYMBOL]: 'GST2',
       [DECIMALS]: 2,
       [EXCHANGE_ADDRESS]: '0x929507CD3D90Ab11eC4822E9eB5A48eb3a178F19'
+    },
+	'0x2b591e99afE9f32eAA6214f7B7629768c40Eeb39': {
```
Comment (PaulStakeHEX) [inline src/contexts/Tokens.js]: This line is indented improperly (2 extra spaces) causing the build to fail.

**Comment by PaulStakeHEX** [comment]
Note the token reputation has been ugpraded on Etherscan to Neutral. :-) https://etherscan.io/token/0x2b591e99afe9f32eaa6214f7b7629768c40eeb39

**Comment by NoahZinsmeister** [comment]
monitoring this in #563

**Comment by Justbeingjustin** [comment]
@NoahZinsmeister ,

How can we get some traction on #563? I know that you mentioned that you are monitoring the issue. Can you please provide an update. We've been patiently waiting and would like a response.

**Pull request closed without merging**